app/repositories/VatSchemeRepository.scala

```diff
@@ -79,6 +82,7 @@ class VatSchemeRepository @Inject() (
   private val acknowledgementRefPrefix = "VRS"
   private val timestampKey             = "timestamp"
   private val internalIdKey            = "internalId"
```

Comment:
Shall we add `private val registrationIdKey = "registrationId"` here as well and then replace any instances of the "timestamp", "internalId" and "registrationId" strings that are still there with these vals.